src/logger/MeetingSessionPOSTLogger.ts

```diff
@@ -105,6 +106,11 @@ export default class MeetingSessionPOSTLogger implements Destroyable {
         const response = await fetch(this.url, {
           method: 'POST',
           body,
+          ...(this.headers
```

Comment:
nit: for readability.. Ignore this commit if the Prettier has automatically fixed your code.

```
...(this.headers ? {
  headers: this.headers
} : {}),
```

Reply:
npm run build:release is fixing that